tests/tests.rs

```diff
@@ -121,7 +121,8 @@ fn run_compile_test(
     style: Option<Style>,
 ) {
     let crate_dir = env::var("CARGO_MANIFEST_DIR").unwrap();
-    let tests_path = Path::new(&crate_dir).join("tests");
+    let tmp_dir = TempDir::new("cbindgen").unwrap();
+    let tests_path = tmp_dir.path();
```

Comment:
This doesn't make sense to me. This breaks updating the test expectations and so on.

Reply:
Ah, I lie, but this still breaks tests that expect to find includes in `tests_path`. Why is this one needed.